packages/ui-text/src/Text/README.md

```diff
@@ -158,3 +176,7 @@ type: example
   This<sup>is</sup> some<sub>text</sub>.
 </Text>
 ```
+
+### DEPRECATED legacy values
+
+Multiple values from `size`, `weight` and `lineHeight` are deprecated, but still supported for backward compatibility reasons. Please only use the above listed, semantic values.
```

Comment:
I think it would be good to list here exactly what is deprecated. (same for Heading). We do not list anywhere the deprecated values

Reply:
Nothing is deprecated for heading. I changed for Text